$NetBSD$

Set LIBS for BSD.
Add LDFLAGS so that the X11 library paths can be used.

--- Makefile.orig	2023-03-21 20:06:13.000000000 +0000
+++ Makefile
@@ -48,7 +48,7 @@ CXX?=g++
 RC?=windres
 
 OURCFLAGS=-g -W -Wall -fno-strict-aliasing -std=c99
-OURCXXFLAGS=-g -W -Wall -fno-exceptions -fno-rtti -std=c++98
+OURCXXFLAGS=-g -W -Wall -fno-exceptions -fno-rtti -std=c++20
 OURCPPFLAGS=-I$(SRC) -I$(ENGINEINC) -I$(MACTROOT) -I$(AUDIOLIBROOT)/include
 OURLDFLAGS=
 
@@ -235,9 +235,11 @@ $(AUDIOLIBROOT)/%:
 
 sw$(EXESUFFIX): $(GAMEOBJS) $(ENGINEROOT)/$(ENGINELIB) $(AUDIOLIBROOT)/$(JFAUDIOLIB)
 	$(CXX) $(CPPFLAGS) $(OURCPPFLAGS) $(CXXFLAGS) $(OURCXXFLAGS) -o $@ $^ $(LDFLAGS) $(OURLDFLAGS)
+	$(CXX) $(CPPFLAGS) $(OURCPPFLAGS) $(CXXFLAGS) $(OURCXXFLAGS) -o $@ $^ $(LDFLAGS) $(OURLDFLAGS) $(PKGSRCLDFLAGS)
 
 build$(EXESUFFIX): $(EDITOROBJS) $(ENGINEROOT)/$(EDITORLIB) $(ENGINEROOT)/$(ENGINELIB)
 	$(CXX) $(CPPFLAGS) $(OURCPPFLAGS) $(CXXFLAGS) $(OURCXXFLAGS) -o $@ $^ $(LDFLAGS) $(OURLDFLAGS)
+	$(CXX) $(CPPFLAGS) $(OURCPPFLAGS) $(CXXFLAGS) $(OURCXXFLAGS) -o $@ $^ $(LDFLAGS) $(OURLDFLAGS) $(PKGSRCLDFLAGS)
 
 # RULES
 $(SRC)/%.$o: $(SRC)/%.c
